res/linux/15-mixxx-usb.rules

```diff
@@ -0,0 +1,11 @@
+# This udev rule allows Mixxx to access HID and USB Bulk controllers when running as a normal user
```

Comment:
Reading up on Debian packaging for udev rules, we aren't supposed to put the priority in the name. Could you rename this file ```mixxx.usb.rules```?